mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/DeletedMessage.java

```diff
@@ -239,7 +240,6 @@ public final boolean equals(Object o) {
                 && Objects.equals(this.messageId, that.messageId)
                 && Objects.equals(this.originMailboxes, that.originMailboxes)
                 && Objects.equals(this.owner, that.owner)
-                && Objects.equals(this.content, that.content)
```

Comment:
see my comment in the other PR.

Reply:
Would https://github.com/linagora/james-project/pull/2194/commits/b454ee74442f412225943e2cc2b34ac7c3eabfd9 satisfy you ?